src/tir/transforms/hoist_if_then_else.cc

```diff
@@ -93,11 +112,33 @@ using HoistForIfTuple = std::tuple<bool, const ForNode*, const IfThenElseNode*>;
  *            if (likely(j > 2))
  *                A[i+j+k] = B[i+j+k]
  *
+ *
+ * This pass do hoisting for Block scope variables also.
```

Comment:
The term "Block scope variables" is a little bit confusing to me. Is it equivalent to say variables defined in `Attr` nodes?

Reply:
Yes. It is referring to Attr nodes.
"Block scope variables" is just for my internal mapping :)